### ISSUE TITLE
Possibility for modded mob support?

### ISSUE THREAD

**Issue by CalaMariGold** [Tschipcraft/spawnanimations#22]
### Describe the solution you'd like

Is mobbed mob support planned for the future?

**Comment by Tschipcraft** [comment]
Yes, this is technically already possible to do by adding any modded entity to the provided tag list, see [exclude or include entities to existing animations](https://github.com/Tschipcraft/spawnanimations/wiki/Exclude-Include-Entities) in the wiki. Actually creating and maintaining support for mods is the other thing. Feel free to suggest mobs from other mods you want to see support for or contribute to this project (right now there is no separate tag file for mod support, which I may want to add in the future to differentiate between vanilla and mods or even different mods)

**Comment by Tschipcraft** [comment]
Fyoncle has taken on the challenge of creating and maintaining mod support with his [Spawn Animations Compats Pack](https://modrinth.com/datapack/ofDka6PS)! 🎉 
See #36
